lib/rules/no-negated-condition.js

```diff
@@ -1,24 +1,144 @@
 import AstNodeInfo from '../helpers/ast-node-info.js';
+import createErrorMessage from '../helpers/create-error-message.js';
 import Rule from './_base.js';
 
 const ERROR_MESSAGE_FLIP_IF =
   'Change `{{if (not condition)}} {{prop1}} {{else}} {{prop2}} {{/if}}` to `{{if condition}} {{prop2}} {{else}} {{prop1}} {{/if}}`.';
 const ERROR_MESSAGE_USE_IF = 'Change `unless (not condition)` to `if condition`.';
 const ERROR_MESSAGE_USE_UNLESS = 'Change `if (not condition)` to `unless condition`.';
+const ERROR_MESSAGE_NEGATED_HELPER = 'Simplify unnecessary negation of helper.';
+
+const DEFAULT_CONFIG = {
+  simplifyHelpers: false,
+};
+
+function isValidConfigObjectFormat(config) {
+  for (let key in DEFAULT_CONFIG) {
+    let value = config[key];
+
+    if (value === undefined) {
+      config[key] = DEFAULT_CONFIG[key];
+    } else if (typeof value !== 'boolean') {
+      return false;
+    }
+  }
+
+  return true;
+}
 
 export default class NoNegatedCondition extends Rule {
+  parseConfig(config) {
+    let configType = typeof config;
+
+    switch (configType) {
+      case 'boolean':
+        return config ? DEFAULT_CONFIG : false;
+      case 'object':
+        if (isValidConfigObjectFormat(config)) {
+          return config;
+        }
+        break;
+      case 'undefined':
+        return false;
+    }
+
+    let errorMessage = createErrorMessage(
+      this.ruleName,
+      [
+        '  * boolean -- `true` for enabled / `false` for disabled\n' +
+          '  * object --\n' +
+          '    *  `simplifyHelpers` -- boolean - simplify negated helpers like `(not (eq ..))` or `(not (gt ...))`, `true` to enable / `false` to disable (default)',
```

Comment:
Update to match suggestion here: https://github.com/ember-template-lint/ember-template-lint/pull/2653/files#r1007167436